src/peergos/shared/user/fs/FileTreeNode.java

```diff
@@ -1003,11 +1006,31 @@ public static FileTreeNode createRoot(TrieNode root) {
         return new byte[0];
     }
 
+    public static byte[] generateVideoThumbnail(byte[] videoBlob) {
```

Comment:
is it possible to do this without creating a random file somewhere? 
also why is .mp4 hard coded?

Reply:
javafx Media class can only read from a file. I wish it read from a memory stream. From testing it seemed the file extension was important.

Reply:
what about .webm or other extensions?

Reply:
javafx currently only supports limited set of video formats